addons/web/static/src/js/core/utils.js

```diff
@@ -729,17 +729,20 @@ var utils = {
      *
      * @param {any[]} array
      * @param {string | function} [criterion]
+     * @param {boolean} [asc=true] sort by ascending if true
      */
-    sortBy: function (array, criterion) {
+    sortBy: function (array, criterion, asc = true) {
```

Comment:
I'd make this
```
sortBy(array, criterion, order='asc') {
.....

return order === 'asc' ? result : -result;
}
```
To make the API more ecplicit

Reply:
done